src/test/java/ValheimTest.java

```diff
@@ -0,0 +1,92 @@
+import io.restassured.response.ValidatableResponse;
+import models.valheim.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import requestprovider.ValheimRequestProvider;
+import utils.RestUtils;
+
+import java.util.Arrays;
+
+public class ValheimTest extends BaseTest {
+
+    private final ValheimRequestProvider requestProvider = new ValheimRequestProvider();
+
+
+    @Test
+    public void ValheimTest() {
+        ValidatableResponse response = RestUtils.get(requestProvider.prepareGetValheimListRequest());
+        response.statusCode(200);
+        ValheimResponse character = response.extract().body().as(ValheimResponse.class);
+        Assertions.assertEquals(getValheimResponse(), character);
```

Comment:
Even so I've proposed these names, now I see that they are not that clear. Maybe you can call expected response method getExpectedValheimHeroResponse, and your actual response as response for example.